src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -450,6 +447,7 @@ InitIonizationModule (const amrex::Geometry& geom, PlasmaParticleContainer* prod
         h_adk_prefactor[i] = dt * wa * C2 * ( Uion/(2*UH) )
             * std::pow(2*std::pow((Uion/UH),3./2)*Ea,2*n_eff - 1);
         h_adk_exp_prefactor[i] = -2./3 * std::pow( Uion/UH,3./2) * Ea;
+        h_laser_adk_prefactor[i] = (3 / MathConst::pi) * std::pow(Uion/UH, -3./2.) / Ea;
```

Comment:
Just to be consistent with usage above.
```suggestion
        h_laser_adk_prefactor[i] = (3. / MathConst::pi) * std::pow(Uion/UH, -3./2.) / Ea;
```